Data/Vector/Fusion/Stream/Monadic.hs

```diff
@@ -1117,6 +1118,20 @@ unfoldrNM m f t = Stream step (t,m)
                                  Nothing     -> Done
                              ) (f s)
 
+unfoldrExactN :: Monad m => Int -> (s -> (a, s)) -> s -> Stream m a
+{-# INLINE_FUSED unfoldrExactN #-}
+unfoldrExactN n f = unfoldrExactNM n (return . f)
+
+-- | Unfold at most @n@ elements with a monadic functions
```

Comment:
Should say `Unfold exactly @n@  elements ...`

Reply:
And I thought I fixed haddock... Thanks! I guess I mixed up copy-paste somewhere

Reply:
The doc is still not right :)

Should say: `Unfold exactly @n@ elements with a monadic function`

(note `function` is not plural)